### DISCUSSION TITLE
같은 QR코드로 접속했는데, 서로 다른 방에 있어요.

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1019] [5. Q&A]
같은 QR인데 왜 서로 볼 수 없는 걸까요..?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/567 이슈와 동일한 내용으로 개선 작업 진행 중에 있습니다.
감사합니다.
